content/src/content/jcr_root/apps/core/wcm/components/teaser/v1/teaser/clientlibs/editor/js/teaser.js

```diff
@@ -107,8 +107,8 @@
 
     function toggleActionItems(actionsMultifield, disabled) {
         actionsMultifield.find("coral-multifield-item").each(function(ix, item) {
-            var linkField = $(item).find("foundation-autocomplete[name='link']").adaptTo("foundation-field");
-            var textField = $(item).find("input[name='text']").adaptTo("foundation-field");
+            var linkField = $(item).find("[data-cmp-teaser-v1-dialog-edit-hook='actionLink']").adaptTo("foundation-field");
```

Comment:
I understand it's out of the scope of the issue, but would be great if we could apply the mechanism to the whole dialog, or at least have a follow up issue to track the effort for all components.

Reply:
see comment above.